docs/experiments/multiple-focus-metavariables.md

```diff
@@ -6,37 +6,47 @@ description: "With this rule, Semgrep matches all pieces of code captured by the
 
 # Using multiple focus metavariables
 
-Semgrep matches all pieces of code captured by focus metavariables when you specify them in a rule. Include the metavariables that you want to focus on by specifying each in a YAML list. This enables Semgrep to match these metavariables regardless of their position in code.
+Semgrep matches all pieces of code captured by focus metavariables when you specify them in a rule. Include the metavariables that you want to focus on by specifying each in a YAML list or by including more focus-metavariables. 
 
 :::info
 This feature is using `focus-metavariable`, see [`focus-metavariable`](/writing-rules/rule-syntax/#focus-metavariable) documentation for more information.  
 :::
 
 There are two methods of focusing Semgrep on multiple metavariables. Each of them can lead to different results:
```

Comment:
```suggestion
```

Reply:
Why not?

Reply:
Oh, I guess the "and"/"advanced" method doesn't actually focus in on multiple, it just focuses in on whichever has a match with a smaller range if their ranges overlap.

Reply:
@enncoded this text is no longer in the document, can we comment on the current state?